config/crds/migration.openshift.io_directvolumemigrations.yaml

```diff
@@ -131,7 +133,6 @@ spec:
                   verify:
                     type: boolean
                 required:
-                - targetAccessModes
```

Comment:
I know that this is a required value on a PVC object. Any reason why we would like to remove this as required from the API?

Reply:
It shouldn't be required. This is only used when access mode is being changed. The only reason it was required before was the "omitEmpty" typo (instead of "omitempty"). Let me verify that where it's used in the code it's not expected, though.

Reply:
Actually, maybe the way DVM is written they're always set here. Hmm. Maybe omitEmpty shouldn't have been set on the CRD?

Reply:
Looks like while MigPlan selection makes AccessModes optional, DVM copies the PVC's access modes if the migplan doesn't override them.